viz.yaml

```diff
@@ -463,6 +464,7 @@ publish:
       county_boundaries_mobile: county_boundaries_mobile_topojson
       county_centroids_wu: county_centroids_wu_tsv
       wu_data_15_range:  wu_data_15_range_json
+      wu_state_data_json: wu_state_data_json
```

Comment:
So I have literally no idea how dev/beta are working without this line.

Reply:
if this file is necessary (looks like it is!) dev/beta may be working because the file was built and put in `target` on an earlier version and then it never ends up getting deleted.